src/doc/reference.md

```diff
@@ -2070,6 +2070,9 @@ macro scope.
    trait of the same name. `{Self}` will be replaced with the type that is supposed
    to implement the trait but doesn't. To use this, the `on_unimplemented` feature gate
    must be enabled.
+- `must_use` - on structs and enums, will warn if this value isn't bound to a variable.
```

Comment:
I would argue that `some_fn(my_var)` is a use under @Manishearth's version. It feels a little vague to put "isn't used" without refining it further, but at the same time trying to define what a "use" is feels like it would get overly wordy. Maybe "will warn when this value isn't used or assigned to a variable"?